src/store/cockpitTypes.ts

```diff
@@ -0,0 +1,6 @@
+// TODO: maybe move to cockpit package?
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type Params = Record<string, any>;
```

Comment:
is it possible to remove this comment and choose different type ?

Reply:
I changed it to `unknown`